### PR TITLE
Fix typo, increase C & C++ accuracy, increase  C++ io performance

### PR DESCRIPTION
**Pull request by 0xB00B**

### CODE REVIEW

File: src/leibniz.cpp [hunk 1]
```diff
@@ -1,29 +1,23 @@
...
-	}
-	
-	pi *= 4;
-	
-	std::cout.precision(17); // set no. of decimal digits (upto 50)
-	std::cout << pi << std::endl; // print pi to console
-	return 0;
+    auto infile = std::fopen("rounds.txt", "r");  // open file
+    std::fscanf(infile, "%u", &rounds);           // read from file
+    std::fclose(infile);                          // close file
```
Comment (workingjubilee) [inline src/leibniz.cpp:11]: :raised_eyebrow: No one writes C++ like this. And even the C shouldn't use `fscanf`.

**Review by workingjubilee — COMMENTED** [review]
If you want to improve IO output speed, maybe you should just use `'\n'` instead:

https://twitter.com/lefticus/status/1565729819077160960

**Comment by 0xB00B** [comment]
> If you want to improve IO output speed, maybe you should just use `'\n'` instead:

That won't make any noticeable difference in this particular case. streams are just inherently slow.

**Comment by 0xB00B** [comment]
@niklas-heer ping!

**Comment by niklas-heer** [comment]
@0xB00B thank you very much for your contribution! :)

**Comment by niklas-heer** [comment]
Also, thank you @workingjubilee for adding your thoughts to the conversation! 👍

**Comment by niklas-heer** [comment]
@0xB00B you now can see the latest results generated through GitHub Actions (not my laptop any more) on this repos [GitHub Page](https://niklas-heer.github.io/speed-comparison/).

plot

**Pull request merged**